beacon-light-client/plonky2/balance_verification/get_balances_input.ts

```diff
@@ -8,56 +8,51 @@ import { KeyPrefix, WorkQueue, Item } from '@mevitae/redis-work-queue';
 import validator_commitment_constants from '../constants/validator_commitment_constants.json';
 import { computeEpochAt } from '@dendreth/utils/ts-utils/ssz-utils';
 import { panic } from '@dendreth/utils/ts-utils/common-utils';
-import { CommandLineOptionsBuilder } from '../cmdline';
-import config from '../common_config.json';
+import commonConfig from '../common_config.json';
 import {
   convertValidatorToValidatorPoseidonInput,
   getZeroValidatorPoseidonInput,
 } from './utils';
 
 const CIRCUIT_SIZE = 8;
-let TAKE: number;
 
-(async () => {
+export type GetBalancesInputConfigRequiredFields = {
+  withdrawCredentials: string;
+};
+
+export type GetBalancesInputConfig = GetBalancesInputConfigRequiredFields & {
+  beaconNodeUrls: string[];
+  slot?: number;
+  take: number;
+  offset?: number;
+  redisHost: string;
+  redisPort: number;
+};
+
+export type ParameterType = GetBalancesInputConfigRequiredFields &
+  Partial<GetBalancesInputConfig>;
+
+function getDefaultBalancesConfig(): Omit<
+  GetBalancesInputConfig,
+  keyof GetBalancesInputConfigRequiredFields
+> {
+  return {
+    beaconNodeUrls: commonConfig['beacon-node'],
+    slot: undefined,
+    take: Infinity,
+    offset: undefined,
+    redisHost: commonConfig['redis-host'],
+    redisPort: Number(commonConfig['redis-port']),
+  };
+}
+
+export async function getBalancesInput(options: ParameterType) {
+  const config = { ...getDefaultBalancesConfig(), ...options };
+
   const { ssz } = await import('@lodestar/types');
+  const redis = new RedisLocal(config.redisHost, config.redisPort);
 
-  const options = new CommandLineOptionsBuilder()
-    .usage(
-      'Usage: -redis-host <Redis host> -redis-port <Redis port> -take <number of validators>',
-    )
-    .withRedisOpts()
-    .option('beacon-node', {
-      alias: 'beacon-node',
-      describe: 'The beacon node url',
-      type: 'array',
-      default: config['beacon-node'],
-      description: 'Sets a custom beacon node url',
-    })
-    .option('slot', {
-      alias: 'slot',
-      describe: 'The state slot',
-      type: 'number',
-      default: undefined,
-      description: 'Fetches the balances for this slot',
-    })
-    .option('take', {
-      alias: 'take',
-      describe: 'The number of validators to take',
-      type: 'number',
-      default: Infinity,
-      description: 'Sets the number of validators to take',
-    })
-    .option('offset', {
-      alias: 'offset',
-      describe: 'Index offset in the validator set',
-      type: 'number',
-      default: undefined,
-    })
-    .build();
-
-  const redis = new RedisLocal(options['redis-host'], options['redis-port']);
-
-  TAKE = options['take'];
+  const withdraw_credentials = config.withdrawCredentials;
```

Comment:
```suggestion
  const withdrawCredentials = config.withdrawCredentials;
```
Why not stick with `camel case`